entrypoint.sh

```diff
@@ -8,15 +8,15 @@ config="${awsDir}/config"
 credentials="${awsDir}/credentials"
 
 mkdir -p "${awsDir}"
-echo -e "[profile default]\noutput = json" >"$config"
+echo -e "[profile $INPUT_AWS_PROFILE]\noutput = json" >>"$config"
```

Comment:
I think we may have to do something like 
```suggestion
echo -e "[profile ${INPUT_AWS_PROFILE:-default}]\noutput = json" >>"$config"
```

but if you have tested it to be working fine, then we're good.